chain/stmgr/stmgr.go

```diff
@@ -62,15 +63,24 @@ type versionSpec struct {
 	atOrBelow      abi.ChainEpoch
 }
 
+type migration struct {
+	upgrade       MigrationFunc
+	preMigrations []PreMigration
+	cache         *nv10.MemMigrationCache
```

Comment:
Do we know how much memory this will use?

Reply:
The cache? Maybe a few hundred bytes per actor. It doesn't cache any actual data, just a mapping of `<actor-address>-<old-state-root>` to `<new-state-root>`.